tests/runner.html

```diff
@@ -16,17 +16,45 @@
 <script>
   var suites = [
     'shady.html',
+    'shady.html?noPatch=true',
+
     'shady-dynamic.html',
+    'shady-dynamic.html?noPatch=true',
```

Comment:
Can we do `suites = suites.concat(suites.map(e => e + '?noPatch=true'))` for most cases?